src/Agent.Listener/MessageListener.cs

```diff
@@ -129,12 +146,15 @@ public sealed class MessageListener : AgentService, IMessageListener
             ArgUtil.NotNull(Session, nameof(Session));
             ArgUtil.NotNull(_settings, nameof(_settings));
             var agentServer = HostContext.GetService<IAgentServer>();
+            int consecutiveErrors = 0; //number of consecutive exceptions thrown by GetAgentMessageAsync
```

Comment:
To elaborate more: "GetAgentMessageAsync" may return "null", which is "no error", on the first error I need to print on the console, on any error (first, second, etc) I need to delay before retry.